### DISCUSSION TITLE
Release date for 22.06 ?

### DISCUSSION THREAD

**Discussion by Romain-Geissler-1A** [moby/moby#44088] [Q&A] [answered]
Hi,

Since couple of month a beta release of 22.06 is available. I initially thought that after this beta, other betas/release candidates would be regularly released, and that the final version would come some weeks after the first beta. However it's been couple of months now, so I start to wonder, what is the expected time frame for the final 22.06 ? Is it still expected in 2022 ?

Cheers,
Romain

**Comment by dimitar-dimitrow** [comment]
Hi, is there any update on 22.06 release date?

**Comment by Romain-Geissler-1A** [comment, marked as answer]
Based on the recent creation of branch 23, I would reply to myself that the release date for Docker 22.06 is "never", and that the next major Docker release is now scheduled for 2023 (no idea when in 2023).
